components/clarinet-sdk/node/tests/simnet-usage.test.ts

```diff
@@ -47,6 +47,11 @@ describe("basic simnet interactions", () => {
     simnet.mineEmptyBlocks(4);
     expect(simnet.blockHeight).toBe(blockHeight + 5);
   });
+  it("can not mine empty stacks block in pre-3.0", () => {
+    expect(() => simnet.mineEmptyStacksBlock()).toThrowError(
+      "use mineEmptyBurnBlock in epoch lower than 3.0"
+    );
+  })
```

Comment:
I think it would make more sense group all of the tests related to burn/stacks chains height in one file. Including this tests and the one from `simnet-usage-epoch3.test.ts`. (eg: `block-heights-handling.test.ts` or something else)